buildkite/scripts/merges-cleanly.sh

```diff
@@ -45,5 +45,6 @@ else
   # Found a conflict
   echo "[ERROR] This pull request conflicts with $BRANCH, please open a new pull request against $BRANCH at this link:"
   echo "https://github.com/MinaProtocol/mina/compare/${BRANCH}...${BUILDKITE_BRANCH}"
+  git status
```

Comment:
The `git diff` command above will already dump all of the details of the merge, and adding this here obfuscates the recommendation from the line above. Recommend closing this PR.